srcpkgs/singularity/INSTALL

```diff
@@ -0,0 +1,10 @@
+case "${ACTION}" in
+post)
+	if [ "$(readlink -f . 2>/dev/null || echo .)" != "/" ]; then
+		exit 1
+	else
+		chown root:_singularity usr/libexec/singularity/bin/starter-suid
+		chmod 4755 usr/libexec/singularity/bin/starter-suid
```

Comment:
Isn't an explicit failure here pretty bad? I think XBPS stops updates if you die with failure (instead of returning 0).

Reply:
True that. Didn't think of that.